ragstack/colbert/cassandra_retriever.py

```diff
@@ -113,9 +115,12 @@ def retrieve(
         doc_futures = []
         for qv in query_encodings:
             # per token based retrieval
-            doc_future = self.vector_store.session.execute_async(
-                self.vector_store.query_colbert_ann_stmt, [list(qv), top_k]
-            )
+            bound_statement = self.vector_store.query_colbert_ann_stmt.bind([list(qv), top_k])
+
+            # top-k queries can only be run with consistency level ONE / LOCAL_ONE / NODE_LOCAL
+            bound_statement.consistency_level = ConsistencyLevel.LOCAL_ONE
```

Comment:
I think it's preferable to fix this problem in CassIO/cassandra driver.

@epinzur for your tests, you can disable the warn logging for that particular logger if you want

For users, we should track this on CassIO probably

Reply:
Stefano is fixing this in CassIO soon. Per jbellis, seems like LOCAL_ONE is the recommended strategy for all cassandra: https://github.com/CassioML/cassio/issues/35#issuecomment-1841724411

I'll close the PR.